frontend/plan/src/components/Cart.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import CartSection from "./CartSection";
+import { toggleCheck } from "../actions";
+import { meetingsContainSection } from "../meetUtil";
+import PropTypes from 'prop-types';
+
+class Cart extends Component {
+    render() {
+        const {courses} = this.props;
+        return (
+            <section
+                style={{
```

Comment:
Not necessarily for this PR, but we will have to standardize how we do styling at some point. Hopefully styled components.